tests/test_flow/conftest.py

```diff
@@ -105,6 +106,27 @@ def gcs_builder(builder, tmp_gcs_url_prefix):
     return builder
 
 
+@pytest.fixture
+def fake_aip_builder(gcs_builder, gcs_url_stem):
+    gcs_builder.set("core__aip_execution__enabled", True)
+    # TODO: This is really a hack to make fake AIP tests work. We need
+    # to  specify a real project for input and output uri for AIP task.
+    # In future, we should change this so that the executor specifies
+    # the input and output uris instead. It could also be indicative of
+    # the fact that maybe we should use the gcs bucket for input and
+    # output uri instead of the AIP project (and use project only for
+    # AIP jobs).
+    project = gcs_url_stem[5:]
```

Comment:
Maybe we could `assert gcs_url_stem.startswith('gs://')` to make this clearer?

Reply:
I initially thought the same thing, but then saw that `gcs_url_stem` fixture already does that. But I can add it again for readability.